dll/win32/browseui/explorerband.cpp

```diff
@@ -147,7 +148,7 @@ HRESULT _ReparsePIDL(LPITEMIDLIST buggyPidl, LPITEMIDLIST *cleanPidl)
 CExplorerBand::CExplorerBand()
     : m_pSite(NULL)
     , m_fVisible(FALSE)
-    , m_bNavigating(FALSE)
+    , m_bNavigating(0)
```

Comment:
Revert this.
Why are you hell-bent on making your changes more difficult to read? Playing compiler makes your code less portable. And while this is not too difficult to follow, it's more difficult for no benefit.